R/display.R

```diff
@@ -2,49 +2,61 @@
 #'
 #' @param message character with any message you want
 #' @param position character with position of the parameter. Default "top right".
+#' @param type character with display type to specify the id. Default to "message"
 #'
-#' @return div which wraps your message to display it in the top-right corner of your shiny app.
+#' @return div which wraps your message to display it in the {position} corner of your shiny app.
 #' @export
 #' @import glue
-display <- function(message, position = "top right") {
+display <- function(message, position = "top right", type = "message") {
   allowed_positions <- c("top right", "top left", "bottom right", "bottom left")
   if (!position %in% allowed_positions)
     stop("Position argument not allowed.")
+  fixed_layout <- "fixed"
+  # when called from inside infoPanel function position arguments are supposed to be overwritten
+  # and css position should be empty
+  plausible_info_panel_call <- sys.call(max(-8, -length(sys.calls()) + 1))
```

Comment:
@dokato I've found that that `sys.call` was causing problems when doing `devtools::check`. I don't know why it worked when running example app manually then. I was aware that this is a "risky" point in the code, but had no other idea how, in the most user friendly way*, recognize if sth is called from inside `info_panel` or not. If you have some please share.

*i.e users use info boxes the same way no matter if they are used inside info_panel or not and do not need to specify position each time.

Reply:
I see... but it looks like this condition is wrong: `max(-8, -length(sys.calls()) + 1)`. When I run it with just `sys.call()` all works just fine.